templates/launch.html

```diff
@@ -326,13 +381,13 @@ <h3>${survey_type} Survey Metadata</h3>
         document.querySelector(`#${el_id}`).value = uuidv4();
     }
 
-    function numericId(el_id) {
+    function numericId() {
         var result = '';
         var chars = '0123456789';
         for (var i = 16; i > 0; --i) {
             result += chars[Math.round(Math.random() * (chars.length - 1))];
         }
-        document.querySelector(`#${el_id}`).value = result;
+        document.querySelector(`#response_id`).value = result;
     }
 
     function ruref(el_id) {
```

Comment:
This function is causing an error in the console because you have removed the `#ru_ref` element. So it, alongside the call for it should probably be removed

Reply:
Removed now